pkg/exporter/process.go

```diff
@@ -215,15 +215,10 @@ func (ep *ExportingProcess) NewTemplateID() uint16 {
 func (ep *ExportingProcess) createAndSendMsg(set entities.Set) (int, error) {
 	// Create a new message and use it to send the set.
 	msg := entities.NewMessage(false)
```

Comment:
Since we have the set, we know the memory size that needs to be allocated right? Can't we use that here?

Reply:
Found that we don't actually append set buffer into message, so the message size is fixed (header length). When sending message, we just combine message header and set buffer. So we don't need to consider message buffer pre-alloc here.

Reply:
Sounds good. Did you see better performance?

Reply:
Sadly no :(